app/models/rr_pending_change.rb

```diff
@@ -14,10 +14,7 @@ def self.last_id
   class << self; alias_method :backlog, :count; end
 
   def self.backlog_details
-    counts = all(
-      :select => "change_table, COUNT(id) AS count_all",
-      :group  => "change_table"
-    )
-    counts.each_with_object({}) { |c, h| h[c.change_table] = c.count_all.to_i }
+    counts = select("change_table, COUNT(id) AS count_all").group("change_table").order('count_all DESC').limit(20).to_a
+    counts.each_with_object([]) { |c, a| a << [c.change_table, c.count_all.to_i] }
```

Comment:
Is this such a bottleneck that it matters?  I'd much rather prefer readbility (particularly in the log message) if it doesn't make a difference.